dao/src/test/java/org/thingsboard/server/dao/sql/user/JpaUserCredentialsDaoTest.java

```diff
@@ -34,41 +38,73 @@
+            userCredentialsDao.removeById(TenantId.SYS_TENANT_ID, userCredentials.getUuidId());
+        }
+    }
+
     @Test
-    @DatabaseSetup("classpath:dbunit/user_credentials.xml")
     public void testFindAll() {
         List<UserCredentials> userCredentials = userCredentialsDao.find(SYSTEM_TENANT_ID);
-        assertEquals(2, userCredentials.size());
+        assertEquals(COUNT_USER_CREDENTIALS + 1, userCredentials.size());
     }
 
     @Test
-    @DatabaseSetup("classpath:dbunit/user_credentials.xml")
     public void testFindByUserId() {
-        UserCredentials userCredentials = userCredentialsDao.findByUserId(SYSTEM_TENANT_ID, UUID.fromString("787827e6-27d7-11e7-93ae-92361f002671"));
+        UserCredentials result = userCredentialsList.get(0);
+        assertNotNull(result);
+        UserCredentials userCredentials = userCredentialsDao.findByUserId(SYSTEM_TENANT_ID, result.getUserId().getId());
         assertNotNull(userCredentials);
-        assertEquals("4b9e010c-27d5-11e7-93ae-92361f002671", userCredentials.getId().toString());
-        assertEquals(true, userCredentials.isEnabled());
-        assertEquals("password", userCredentials.getPassword());
-        assertEquals("ACTIVATE_TOKEN_2", userCredentials.getActivateToken());
-        assertEquals("RESET_TOKEN_2", userCredentials.getResetToken());
+        assertEquals(result.getId(), userCredentials.getId());
+        assertEquals(result.isEnabled(), userCredentials.isEnabled());
+        assertEquals(result.getPassword(), userCredentials.getPassword());
+        assertEquals(result.getActivateToken(), userCredentials.getActivateToken());
+        assertEquals(result.getResetToken(), userCredentials.getResetToken());
     }
 
     @Test
-    @DatabaseSetup("classpath:dbunit/user_credentials.xml")
     public void testFindByActivateToken() {
-        UserCredentials userCredentials = userCredentialsDao.findByActivateToken(SYSTEM_TENANT_ID, "ACTIVATE_TOKEN_1");
+        UserCredentials userCredentials = userCredentialsDao.findByActivateToken(SYSTEM_TENANT_ID, ACTIVATE_TOKEN);
         assertNotNull(userCredentials);
-        assertEquals("3ed10af0-27d5-11e7-93ae-92361f002671", userCredentials.getId().toString());
+        UserCredentials result = userCredentialsList.get(0);
+        assertNotNull(result);
+        assertEquals(result.getId(), userCredentials.getId());
     }
 
     @Test
-    @DatabaseSetup("classpath:dbunit/user_credentials.xml")
     public void testFindByResetToken() {
-        UserCredentials userCredentials = userCredentialsDao.findByResetToken(SYSTEM_TENANT_ID, "RESET_TOKEN_2");
+        UserCredentials userCredentials = userCredentialsDao.findByResetToken(SYSTEM_TENANT_ID, RESET_TOKEN);
         assertNotNull(userCredentials);
-        assertEquals("4b9e010c-27d5-11e7-93ae-92361f002671", userCredentials.getId().toString());
+        UserCredentials result = userCredentialsList.get(0);
```

Comment:
this is a little bit confusing to see the variable `result` with the save `userCredentialsList.get(0);`
` UserCredentials result = userCredentialsList.get(0);`
I suggest you to put another variable at the beginning of this class and assign a valuable name for that